packages/pages/src/vite-plugin/build/closeBundle/bundleValidator.ts

```diff
@@ -0,0 +1,51 @@
+import { statSync } from "fs";
+import glob from "glob";
+import path from "path";
+
+const PLUGIN_FILESIZE_LIMIT = 1.5; // MB
+const PLUGIN_TOTAL_FILESIZE_LIMIT = 10; // MB
+
+/**
+ * Validates the bundled files.
+ */
+export const validateBundles = () => {
```

Comment:
this isn't all the bundles -- just the server-side bundles. should that be noted?